doc/source/io.rst

```diff
@@ -225,9 +225,8 @@ NA and Missing Data Handling
 
 na_values : scalar, str, list-like, or dict, default ``None``
   Additional strings to recognize as NA/NaN. If dict passed, specific per-column
-  NA values. By default the following values are interpreted as NaN:
-  ``'-1.#IND', '1.#QNAN', '1.#IND', '-1.#QNAN', '#N/A N/A', '#N/A', 'N/A', 'n/a', 'NA',
-  '#NA', 'NULL', 'null', 'NaN', '-NaN', 'nan', '-nan', ''``.
+  NA values. By default the following values are interpreted as NaN: See :ref:`na values const
```

Comment:
The sentence "By default the following values are interpreted as NaN:" can be removed. I think

    See :ref:`na values const <io.navaluesconst>` below for a list of the values interpreted as NaN by default.